nncf/common/pruning/utils.py

```diff
@@ -278,8 +274,6 @@ def count_flops_and_weights_per_node(graph: NNCFGraph,
     Counts the number weights and FLOPs per node in the model for convolution and fully connected layers.
 
     :param graph: NNCFGraph.
-    :param input_shapes: Dictionary of input dimension shapes for convolutions and
-        fully connected layers. E.g {node_name: (height, width)}
     :param output_shapes: Dictionary of output dimension shapes for convolutions and
         fully connected layers. E.g {node_name: (height, width)}
```

Comment:
It seems that `output_shapes` variable also no longer needed.

Reply:
`output_shapes` is needed for computing flops
https://github.com/openvinotoolkit/nncf/blob/d5e9edf8f335fc6d97825ddc07cfbd03594defd2/nncf/common/pruning/utils.py#L321